src/context/WalletProvider/WalletViewsSwitch.tsx

```diff
@@ -88,7 +89,8 @@ export const WalletViewsSwitch = () => {
     history.goBack()
     // If we're back at the select wallet modal, remove the initial route
     // otherwise clicking the button for the same wallet doesn't do anything
-    if (history.location.pathname === INITIAL_WALLET_MODAL_ROUTE) {
+    const { pathname } = history.location
+    if ([INITIAL_WALLET_MODAL_ROUTE, NativeWalletRoutes.Load].includes(pathname)) {
```

Comment:
As you changed every paths to an enum in this PR, can we also move `INITIAL_WALLET_MODAL_ROUTE` to `WalletProvider/config.ts`? I was thinking about doing it at the same time than migrating to this enum, because we would need to consume it in multiple places

![image](https://github.com/shapeshift/web/assets/14963751/bc7b72f3-7970-42b5-805e-aa3618bc85d8)

Reply:
Maybe we should do it in a follow up?

Reply:
Agree, though I'd say it's fine where it is right now (for this PR) as it's only used on one place. However, we should move it if/when we start consuming it in multiple places.

Reply:
Oh, you're saying that the logic _is_ actually used elsewhere, though we are only consuming the constant in one place - yes, we should definitely fix this (though not for this PR).